cmd/kube-controller-manager/app/rbac.go

```diff
@@ -19,14 +19,10 @@ package app
 import (
 	"net/http"
 
-	"k8s.io/apimachinery/pkg/runtime/schema"
 	"k8s.io/kubernetes/pkg/controller/clusterroleaggregation"
 )
 
 func startClusterRoleAggregrationController(ctx ControllerContext) (http.Handler, bool, error) {
-	if !ctx.AvailableResources[schema.GroupVersionResource{Group: "rbac.authorization.k8s.io", Version: "v1", Resource: "clusterroles"}] {
```

Comment:
I don't think the RBAC APIs are required for conformance, can you link me?

Reply:
I have added this judgment back